Sources/Foundation/NSStringAPI.swift

```diff
@@ -297,6 +212,7 @@ extension String {
     /// frees the buffer.
     ///
     /// - Warning: This initializer is not memory-safe!
+    @available(swift, deprecated: 6.0, message: "String does not support no-copy initialization")
```

Comment:
These (and the other deprecated functions) are now deprecated on Darwin, so this updates these to match Darwin